cupy/testing/array.py

```diff
@@ -128,6 +128,48 @@ def assert_array_list_equal(xlist, ylist, err_msg='', verbose=True):
             verbose=verbose)
 
 
+def assert_array_exactly_equal(x, y, err_msg='', verbose=True):
```

Comment:
How about adding `strides_check` option in `assert_array_equal`, instead of creating a new function?